rotkehlchen/data_migrations/migrations/migrations_18.py

```diff
@@ -221,4 +221,9 @@ def _remove_manualcurrent_oracle(rotki: 'Rotkehlchen') -> None:
                 ),
             )
 
+    @progress_step(description='Cleaning up Yearn cache.')
+    def _remove_yearn_cache(rotki: 'Rotkehlchen') -> None:
+        with GlobalDBHandler().conn.write_ctx() as write_cursor:
+            write_cursor.execute('DELETE FROM unique_cache WHERE key=?', ('YEARN_VAULTS',))
```

Comment:
why clean up the cache for yearn? You just removed an api source.

Reply:
Because we store the total amount of entries and this was `len(old_api) + len(new_api)`. To prevent any issue we force a refresh but sicne the tokens should already be in the globaldb it will be fast. Just loop the vaults